test/writer/drift_model_writer/enum_field/output.txt

```diff
@@ -0,0 +1,45 @@
+// GENERATED CODE - DO NOT MODIFY BY HAND
+
+import 'package:drift/drift.dart';
+import 'package:model_generator_example/base_dir/path_to_my_model/person.dart';
+import 'package:model_generator_example/database/model_generator_example_database.dart';
+import 'package:model_generator_example/model/user/person/gender.dart';
+
+@DataClassName('DbPerson')
+class DbPersonTable extends Table {
+  TextColumn get firstName => text()();
+
+  TextColumn get prefferedGender => text().map(const PersonTableGenderConverter())();
+
+}
+
+extension DbPersonExtension on DbPerson {
+  Person get model => Person(
+        firstName: firstName,
+        prefferedGender: prefferedGender,
+      );
+}
+
+extension PersonExtension on Person {
+  DbPerson get dbModel => DbPerson(
+        firstName: firstName,
+        prefferedGender: prefferedGender,
+      );
+}
+
+class PersonTableGenderConverter extends TypeConverter<Gender, String> {
+  const PersonTableGenderConverter();
+
+  @override
+  Gender fromSql(String fromDb) {
+    for (final value in Gender.values) {
+      if (value.toString() == fromDb) return value;
```

Comment:
Good point, this can be solved if we did something else I'd like to see. I suggest we add the JSONValue to the enum as a field, this way we can much more easily convert between the two and can also use that value to do the check here :-) WDYT

Reply:
Fixed in #129